Microsoft.Identity.Web/Client/TokenAcquisition.cs

```diff
@@ -155,110 +137,55 @@ public async Task AddAccountToCacheFromAuthorizationCode(AuthorizationCodeReceiv
 
         /// <summary>
         /// Typically used from an ASP.NET Core Web App or Web API controller, this method gets an access token
-        /// for a downstream API on behalf of the user account which claims are provided in the <see cref="HttpContext.User"/>
-        /// member of the <paramref name="context"/> parameter
+        /// for a downstream API on behalf of the user account
         /// </summary>
         /// <param name="context">HttpContext associated with the Controller or auth operation</param>
         /// <param name="scopes">Scopes to request for the downstream API to call</param>
-        /// <param name="tenantId">Enables to override the tenant/account for the same identity. This is useful in the
-        /// cases where a given account is guest in other tenants, and you want to acquire tokens for a specific tenant</param>
         /// <returns>An access token to call on behalf of the user, the downstream API characterized by its scopes</returns>
-        public async Task<string> GetAccessTokenOnBehalfOfUser(HttpContext context, IEnumerable<string> scopes, string tenant = null)
```

Comment:
Again, there are scenarios where the tenant is used.
Do we suggest to have code without using Microsoft.Identity.Web ?

Reply:
right...i don't know how much Microsoft.identity.Web is used. Can just write a helper function to handle the OBO call. we can discuss

---
In reply to: [318912996](https://github.com/Azure-Samples/active-directory-dotnet-native-aspnetcore-v2/pull/93#discussion_r318912996) [](ancestors = 318912996)